tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title must be at least 5 characters.')
+                    ->assertDontSee('Lốc bụi');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.3: Create blog: Fill all input but fill in title over 100 characters
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogFillTitleOver100Character()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            //string over 100 character
+            $textOver100 = str_repeat('a', 101);
+
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', $textOver100)
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title may not be greater than 100 characters.')
+                    ->assertDontSee($textOver100);
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.4: Create blog: Do not fill input
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogDoNotFillInput()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title field is required.')
+                    ->assertSee('The content field is required.');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.5: - Create blog: Content input empty
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogContentInputEmpty()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'new title')
```

Comment:
this content and the once in Test Case report is different , why ?

Reply:
I fixed in file BlogManageTest.php